src/forms/css/forms.css

```diff
@@ -304,3 +304,8 @@ since IE8 won't execute CSS that contains a CSS3 selector.
     color: #666;
     font-size: 0.875em;
 }
+
+/* File Upload Label */
+.pure-form label.pure-button {
+    margin: 0;
+}
```

Comment:
This fixes the margin issue.

Reply:
without this, what does it look like?

Reply:
The buttons are misaligned:
<img width="174" alt="Screen Shot 2020-09-07 at 2 49 20 PM" src="https://user-images.githubusercontent.com/1103622/92414331-5c1b7080-f119-11ea-8ad1-a28ed9f898f1.png">